casper/src/main/scala/coop/rchain/casper/util/SignatureAlgorithms.scala

```diff
@@ -4,10 +4,19 @@ import coop.rchain.crypto.signatures.{Ed25519, Secp256k1}
 object SignatureAlgorithms {
   type SignatureAlgorithm = (Array[Byte], Array[Byte]) => Array[Byte]
   @SuppressWarnings(Array("org.wartremover.warts.Throw")) // TODO remove throw
-  def lookup(algorithm: String): SignatureAlgorithm =
+  def lookupSigAlg(algorithm: String): SignatureAlgorithm =
     algorithm match {
       case "ed25519"   => Ed25519.sign
       case "secp256k1" => Secp256k1.sign
       case _           => throw new Exception(s"Unknown signature algorithm $algorithm")
     }
+
+  type SignatureVerifyAlgorithm = (Array[Byte], Array[Byte], Array[Byte]) => Boolean
+  @SuppressWarnings(Array("org.wartremover.warts.Throw")) // TODO remove throw
+  def lookupSigVerifyAlg(algorithm: String): SignatureVerifyAlgorithm =
```

Comment:
So where do you use this?

Reply:
This was part of a larger PR at first which used it. It should be removed.